survey/locales/en/survey.yaml

```diff
@@ -100,21 +100,35 @@ homepage:
         <p class="left">We thank you for your contribution to the planning of
         tomorrow's transportation!</p><br/> <p class="left _strong"
         style="margin-bottom: 1rem;">The deadline to participate is December
-        16th, 2024.</p><br/><p class="left"><em>For more information about the
+        16th, 2025.</p><br/><p class="left"><em>For more information about the
         Survey, please visit the following link</em>: <a
         href="https://www.transports.gouv.qc.ca/fr/ministere/Planification-transports/enquetes-origine-destination/Pages/enquetes-origine-destination.aspx"
         target="_blank">Enquêtes origine-destination - Transports et Mobilité
         durable Québec (gouv.qc.ca)</a>.</em></p><p class="center _oblique"
-        style="margin: 2rem 0;"><a href="mailto:eod@transports.gouv.qc.ca"
+        style="margin: 2rem 0;"><a href="mailto:EOD@transports.gouv.qc.ca"
         target="_blank">Contact us</a></p>
     start: Start
 footer: >-
     <div style="display: flex;justify-content: center;"><div style="font-size:
     80%;max-width: 700px;margin: 15px;"><p><a
     href="/dist/documents/demo_information_consent_form.pdf"
     target="_blank">Information and Consent Form</a> | <a
-    href="mailto:mobilite@polymtl.ca?subject=Demo%20Survey"
+    href="mailto:EOD@transports.gouv.qc.ca?subject=Origin-Destination%20Survey"
```

Comment:
selon #241, il faut enlever le Form de Information and Consent Form en haut. Aussi est-ce que le document pointe à la bonne place? demo_information_consent_form.pdf ne me semble pas correct

Reply:
SVP cérer un autre issue pour vérifier avec Poly pour le nouveau formulaire

Reply:
J'ai retiré le form de consent form